uidefinitions/createUiDefinition.json

```diff
@@ -1301,7 +1297,7 @@
                 "name": "maxNodes",
                 "type": "Microsoft.Common.TextBox",
                 "label": "Max number of nodes",
-                "defaultValue": "1",
+                "defaultValue": "100",
```

Comment:
This default value is pretty large for most of customers, especially for GPU. I would reduce this to something like 16 or 8